### PR TITLE
Create BambooTracker.1

### PR DESCRIPTION
**Pull request by alexmyczko**
a basic first manual page, feel free to enhance as needed.

### CODE REVIEW

File: BambooTracker.1 [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.TH BambooTracker 1 "December 15 2018"
+.SH NAME
+BambooTracker \- YM2608 (OPNA) tracker
+.SH SYNOPSIS
+.B BambooTracker
+.RI " files" ...
+.br
+.SH DESCRIPTION
+This manual page documents briefly the
```
Comment (OPNA2608) [inline BambooTracker.1]:
```suggestion
This manual page briefly documents the
```
or
```suggestion
This manual page quickly summarizes the
```
would be better / more correct.

**Comment by OPNA2608** [comment]
I think it'd make sense to document all/most of the key commands described in the [README](https://github.com/rerrahkr/BambooTracker/blob/86428063251b90a859633b1e8adf5da1215f2a8c/README.md), especially since there isn't much else to say in a man page about a switchless GUI application imo.

**Comment by alexmyczko** [comment]
I agree, however to have a debian package, it is required to have one:
https://www.debian.org/doc/debian-policy/ch-docs.html
although many don't (it is considered a bug):
https://lintian.debian.org/tags/binary-without-manpage.html

**Comment by rerrahkr** [comment]
Thanks for your contributions!
I really do appreciate everyone's help as I do not know much about Linux.

File: BambooTracker.1 (hunk 1 above)
Comment (rerrahkr) [reply BambooTracker.1]: I will fix it when I edit other documents.

Comment (rerrahkr) [reply BambooTracker.1]: fixed 7a934324d9e569d2e62200cad954dda2c148f4fd

**Pull request merged**